submissions/Shrai-dev/html-css-popup/main.css

```diff
@@ -0,0 +1,178 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-align: center;
+}
+
+img {
+    vertical-align: middle;
+}
+
+li {
+    list-style-type: none;
+}
+
+.nav__menu {
+    margin: 25px 0;
+    align-items: flex-end;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    height: 50px;
+}
+
+.nav__menu-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.nav__menu-item {
+    position: relative;
+    margin: 5px 20px;
+}
+
+a:hover,
```

Comment:
good practice - add the classes for links, images (do not add styles to the tags). Could you please fix it?

Reply:
Fixed.